core/src/main/java/org/jruby/ir/targets/IRBytecodeAdapter6.java

```diff
@@ -589,9 +589,9 @@ public void invokeOtherOneFloat(String file, int line, String name, double flote
         if (!MethodIndex.hasFastFloatOps(name)) {
             pushFloat(flote);
             if (callType == CallType.NORMAL) {
-                invokeOther(file, line, name, 1, false, false);
+                invokeOther(file, line, name, 1, false, false, false);
```

Comment:
Yeah I don't stacking booleans like this. I might combine the two into an enum representing what exactly was passed for block argument.